Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -286,6 +455,57 @@
     outputCommands = cms.untracked.vstring('drop *'),
     splitLevel = cms.untracked.int32(0),
 )
+FEVTSIMEventContent.outputCommands.extend(RAWEventContent.outputCommands)
```

Comment:
This shold be better defined extending the FEVTEventContent
Then, whatever already added on top of RAWEventContent to obtain the FEVTEventContent can be removed from the list below